_i18n/fr.yml

```diff
@@ -54,38 +55,370 @@ titles:
+
+mining:
+  translated: "yes"
+  intro1: Monero est une cryptomonnaie qui s'appuie sur une extraction minière par preuve de travail (Proof-of-Work) pour permettre un consensus distribué. Vous trouvez ci-dessous des ressources et des informations vous permettant de commencer à miner.
+  intro2: Le projet Monero ne plébiscite aucun pool, logiciel ou matériel particulier. Le contenu suivant n'est fournit qu'à titre purement informatif.
+  support: Assistance
+  support_para1: Parcourrez
+  support_para2: les Rencontres,
+  support_para3: /r/moneromining (English)
+  support_para4: et
+  pools: Pools
+  pools_para1: Une liste de pools Monero de confiance est disponible
+  pools_para2: ici.
+  benchmarking: Capacités du matériel
+  benchmarking_para1: Vous trouverez ici
+  benchmarking_para2: une liste de GPUs/CPUs et leurs taux de hachage respectifs.
+  software: Mineur
+  software_para: Veuillez noter que certains mineurs pourraient soustraire des frais de développement.
+
+using:
+  translated: "no"
+  intro: Les transactions sont simples avec Monero. Cette page est conçue pour permettre aux utilisateurs de les réaliser.
+  learn: 1. Apprendre
+  learn_para1: Monero est une cryptomonnaie sécurisée, confidentielle et intraçable. Les développeurs et la communauté s'engagent à protéger ces valeurs. Pour en savoir plus, lisez la page
+  learn_para2: Qu'est-ce que Monero
+  learn_para3: Le
+  learn_para4: code source
+  learn_para5: est également disponible pour examen et discussion.
+  support: 2. Demander de l'Assistance
+  support_para1: Il y a une large communauté de soutien qui vous aidera si vous rencontrez quelconques difficultés. Allez voir la page
+  support_para2: Rencontres
+  support_para3: pour plus d'informations.
+  generate: 3. Créer un portefeuille
+  generate_para1: Vous devez posséder un portefeuille Monero pour sécurisé vos fonds. Regardez la page
+  generate_para2: Téléchargements
+  generate_para3: pour trouver une liste des portefeuilles disponibles.
+  generate_para4: The easiest way to run a Monero node, without affecting your home bandwidth, is to purchase a VPS (Virtual Private Server). We strongly recommend
+  generate_para5: using the
+  generate_para6: coupon code to get a discount over and above their already cheap $6/month VPS. Using this coupon code and/or
+  generate_para7: our affiliate link
+  generate_para8: will also assist in the ongoing funding of Monero development.
```

Comment:
```yml
  generate_para4: La façon la plus simple de faire fonctionner un nœud Monero, sans affecter votre bande passante, est de louer un VPS (Virtual Private Server). Nous recommandons fortement d'utiliser 
  generate_para5: avec le code de réduction
  generate_para6: afin d'obtenir une réduction supplémentaire sur leur tarif déjà abordable de VPS à 6$ par mois. En utilisant ce code de réduction et/ou
  generate_para7: notre lien affilié
  generate_para8: vous aiderez également au développement continue de Monero.
```

Reply:
I don't know what @rehrar thinks here, but it's probably better to make a new PR to translate the missing parts, so this PR can stay as clean as possible. But that's just my opinion

Reply:
I have no preference here. Up to @rehrar.